composer.json

```diff
@@ -17,7 +17,7 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": "~2.3|~3.0",
+        "symfony/framework-bundle": "~2.6|~3.0",
```

Comment:
Don't mind that comment, I probably did not see this was supposed to be major in the first place